aqueduct_client/examples/Sample Aqueduct Usage.ipynb

```diff
@@ -0,0 +1,533 @@
+       "      <th>sid</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th rowspan=\"5\" valign=\"top\">2015-12-31</th>\n",
+       "      <th>2</th>\n",
+       "      <td>1.401180</td>\n",
+       "      <td>2.237009</td>\n",
+       "      <td>0.578636</td>\n",
+       "      <td>-0.826188</td>\n",
+       "      <td>-0.685427</td>\n",
+       "      <td>-1.867208</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>24</th>\n",
+       "      <td>-0.303392</td>\n",
+       "      <td>0.143422</td>\n",
+       "      <td>1.312154</td>\n",
+       "      <td>0.256917</td>\n",
+       "      <td>2.293536</td>\n",
+       "      <td>0.158758</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>62</th>\n",
+       "      <td>-0.391409</td>\n",
+       "      <td>-0.700045</td>\n",
+       "      <td>-0.144533</td>\n",
+       "      <td>0.035006</td>\n",
+       "      <td>1.230461</td>\n",
+       "      <td>-0.050641</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>67</th>\n",
+       "      <td>0.447439</td>\n",
+       "      <td>-1.580643</td>\n",
+       "      <td>-0.173034</td>\n",
+       "      <td>0.743537</td>\n",
+       "      <td>-0.707037</td>\n",
+       "      <td>0.145239</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>76</th>\n",
+       "      <td>-0.130972</td>\n",
+       "      <td>-1.241548</td>\n",
+       "      <td>-0.363105</td>\n",
+       "      <td>-0.762339</td>\n",
+       "      <td>-0.446453</td>\n",
+       "      <td>0.997855</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                 ann_vol        ey      fcfy       gpa  log_mcap    lt_mom\n",
+       "date       sid                                                            \n",
+       "2015-12-31 2    1.401180  2.237009  0.578636 -0.826188 -0.685427 -1.867208\n",
+       "           24  -0.303392  0.143422  1.312154  0.256917  2.293536  0.158758\n",
+       "           62  -0.391409 -0.700045 -0.144533  0.035006  1.230461 -0.050641\n",
+       "           67   0.447439 -1.580643 -0.173034  0.743537 -0.707037  0.145239\n",
+       "           76  -0.130972 -1.241548 -0.363105 -0.762339 -0.446453  0.997855"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "result.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition to using the methods highlighted above when creating new outputs, historical point-in-time Aqueduct submissions can also be accessed via the `get_all_pipeline_executions` method."
```

Comment:
--> "In addition to submitting new pipeline runs to Aqueduct, the client also gives you access to the results of all Aqueduct runs you have submitted in the past. To get a list of all past results, use the `get_all_pipeline_executions` method."